src/controllers/PostsController.ts

```diff
@@ -1,24 +1,41 @@
-import { Controller, Get, PathParams, UseBefore } from '@tsed/common';
+import { BodyParams, Controller, Get, PathParams, Post, UseBefore } from '@tsed/common';
 import { Posts } from '../entities/Posts';
+import { Subreddits } from '../entities/Subreddits';
 import { AuthMiddleware } from '../middlewares/AuthMiddleware';
 
 import { PostsService } from '../services/PostsService';
+import { SubredditsService } from '../services/SubredditsService';
 
 
 @Controller('/')
-export class PostsController {
-	constructor(private postsService: PostsService) {}
+export class UserController {
+	constructor(private postsService: PostsService, private subredditsService: SubredditsService) {}
 
 	@Get('/feed')
 	@UseBefore(AuthMiddleware)
 	getPostsFeed(): Promise<Posts[]> {
 		return this.postsService.findAll();
 	}
 
+	@Get('/subreddits')
+	@UseBefore(AuthMiddleware)
+	getSubreddits(): Promise<Subreddits[]> {
+		return this.subredditsService.findAll();
+	}
```

Comment:
this should probably be in SubredditController

Reply:
Actually moved and removed, as Andy did the same function :)